components/common/shoppingCart/ShoppingCartItemRow.vue

```diff
@@ -1,34 +1,44 @@
 <template>
   <div
     ref="hoverRef"
-    class="is-flex is-justify-content-space-between background-hover">
-    <div class="is-flex pr-2">
-      <nuxt-link
-        :to="`/${urlPrefix}/gallery/${nft.id}`"
-        @click.native="emit('click-item')">
-        <BasicImage
-          :src="avatar"
-          :alt="nft?.name"
-          class="border image is-48x48" />
-      </nuxt-link>
-
-      <div
-        class="is-flex is-flex-direction-column is-justify-content-space-between ml-4 limit-width">
+    class="is-flex is-justify-content-space-between background-hover gap-8">
+    <div class="pr-2 w-full no-wrap is-clipped ellipsis">
+      <div class="is-flex">
         <nuxt-link
           :to="`/${urlPrefix}/gallery/${nft.id}`"
-          class="has-text-weight-bold has-text-color line-height-1 no-wrap is-clipped ellipsis"
           @click.native="emit('click-item')">
-          {{ nft.name }}
+          <BasicImage
+            :src="avatar"
+            :alt="nft?.name"
+            class="border image is-48x48" />
         </nuxt-link>
-        <div class="line-height-1 no-wrap is-clipped ellipsis">
-          {{ nft.collection?.name || nft.collection.id }}
+        <div
+          class="is-flex is-flex-direction-column is-justify-content-space-between ml-4 min-width w-full">
+          <nuxt-link
+            :to="`/${urlPrefix}/gallery/${nft.id}`"
+            class="has-text-weight-bold has-text-color line-height-1 no-wrap is-clipped ellipsis"
+            @click.native="emit('click-item')">
+            {{ nft.name }}
+          </nuxt-link>
+          <div class="is-flex is-justify-content-space-between gap-8 w-full">
+            <div class="line-height-1 no-wrap is-clipped ellipsis">
+              {{ nft.collection?.name || nft.collection.id }}
+            </div>
+
+            <div
+              :class="{
+                'line-height-1': true,
```

Comment:
class="line-heigh-1"

Reply:
I use this style because of the conditional hidden class.
I would leave this comment unresolved till we conclude the conversation above.

Reply:
How about this way?
```
:class="[
    {
      conditional-class: boolean
    },
    'line-height-1',
  ]"
```

Reply:
That works as well.
I have made the change